qutebrowser/completion/models/miscmodels.py

```diff
@@ -111,6 +111,27 @@ def __init__(self, match_field='url', parent=None):
                 match_field))
 
 
+class BookmarkCompletionModel(base.BaseCompletionModel):
```

Comment:
Removing this code duplicate will be possible if we can refactor the quickmark and bookmark managers to use a common base class.
See this comment https://github.com/The-Compiler/qutebrowser/pull/681#issuecomment-120668594.